README.md

```diff
@@ -130,7 +130,8 @@ to `./runner.yml` and overriding relevant properties.
 
 This command will also:
 
-- Symlink the theme in  `./build/modules/custom/oe_authentication` so that it's available for the test site
+- **Please note:** project files and directories are symlinked within the test site by using the
+  [OpenEuropa Task Runner's Drupal project symlink](https://github.com/openeuropa/task-runner-drupal-project-symlink) command.
```

Comment:
Why we can't use same information like provided here: https://github.com/openeuropa/open_vocabularies/pull/28/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R45-R57